05_syntax_lab/02.py

```diff
@@ -1,6 +1,17 @@
-"""
-Write a Python program that randomizes 7 numbers
-and prints their sum total.
-If the sum is divisable by 7, also print the word "Boom"
-"""
+#"""
+#Write a Python program that randomizes 7 numbers
+#and prints their sum total.
+#If the sum is divisable by 7, also print the word "Boom"
+#"""
 
+from random import randint
+sum = 0
+for i in range(0,10):
+    num = randint(1,100)
+    print "generated num:" ,num
+    sum+=num
+    
+
+print "sum is " ,sum
```

Comment:
Oh no too many spaces :) print already writes a space between its arguments so no need for the space inside quotes. A better alternative BTW is to use str.format or %